src/eba_db.ml

```diff
@@ -117,7 +120,7 @@ let activation_table :
     Sql.view =
   <:table< activation (
        activationkey text NOT NULL,
-       userid bigint NOT NULL,
+       email text NOT NULL,
```

Comment:
Maybe add an int ID to this table, and use that for referencing its entries from elsewhere?